discord/member.py

```diff
@@ -750,6 +761,7 @@ async def edit(
         roles: Collection[discord.abc.Snowflake] = MISSING,
         voice_channel: Optional[VocalGuildChannel] = MISSING,
         timed_out_until: Optional[datetime.datetime] = MISSING,
+        bypasses_verification: bool = MISSING,
```

Comment:
```suggestion
        bypass_verification: bool = MISSING,
```

I think verb-like wording is better. The flag itself doesn't have to change (ex: `suppress=True` in `Message.edit` but `MessageFlags.suppress_embeds`).

Be sure to change the verbiage elsewhere in the method.